neuralprophet/df_utils.py

```diff
@@ -217,7 +217,7 @@ def data_params_definition(
                 raise ValueError(f"Regressor {reg} not found in DataFrame.")
             data_params[reg] = get_normalization_params(
                 array=df[reg].values,
-                norm_type=config_regressors[reg].normalize,
+                norm_type=config_regressors[reg].normalize if len(df[reg].unique()) > 1 else "off",
```

Comment:
are we operating over all panel time series or on a single one here?

Reply:
Do you mind explaining the reason for this change?
Thank you!

Reply:
@ourownstory  Sure. Usually, we would normalize the data in the future reg column. This will usually throw an error. With a time series that only contains unique values (which we now enable as long as it's not the case for every ID), we would like to skip normalization though. This is on single time series level here.

Reply:
Maybe I am just confused. Here are my thoughts:

If local normalization is enabled, this would still be an issue - we would need to ignore the variable on a local level.

If it's globally normalized, it would be ok, but then we would not need to switch it off on a local level?

Reply:
@judussoari if local normalization is enabled, even if doing global modeling, we still need to throw and error. 
Only special case is Global normalization and global modeling - then a series may have a single value as long as all series together have more than one value.